helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -381,6 +381,9 @@ receivers:
     # use the read-only endpoint instead.
     auth_type: none
     endpoint: ${K8S_NODE_IP}:10255
+    {{ else if eq .Values.distribution "aks" }}
+    ca_file: "/hostfs/etc/kubernetes/certs/kubeletserver.crt"
+    endpoint: ${K8S_NODE_NAME}:10250
```

Comment:
```suggestion
    auth_type: serviceAccount
    endpoint: ${K8S_NODE_NAME}:10250
```

Do we need to add the auth_type like we have in the else bit? Or maybe this is the default value?